college_flow_app/lib/features/list_subjects/widgets/subject_tile.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:college_flow_app/config/design_system/data/colors/colors.dart';
+import 'package:college_flow_app/config/design_system/data/object_styles/object_styles.dart';
+import 'package:college_flow_app/config/design_system/data/spacing/spacing.dart';
+import 'package:college_flow_app/config/routes/flow_routes.dart';
+import 'package:college_flow_app/features/review/presentation/review_page.dart';
+import 'package:college_flow_app/shared/widgets/flow_icon.dart';
+import 'package:flutter/material.dart';
+
+import '../../review/list_example.dart';
+
+class SubjectTile extends StatelessWidget {
+  const SubjectTile({
+    Key? key,
+    required this.subject,
+  }) : super(key: key);
+
+  final Subject subject;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+      visualDensity: const VisualDensity(vertical: -4),
+      horizontalTitleGap: 16.0,
```

Comment:
on horizontalTitleGap use ds token. is called spacing I think is `spacingXXXS` or something like this